engine/table/src/main/java/io/deephaven/engine/util/config/MutableInputTable.java

```diff
@@ -20,7 +20,7 @@
  * Implementations of this interface will make their own guarantees about how atomically changes will be applied and
  * what operations they support.
  */
-public interface MutableInputTable extends InputTableRowSetter, InputTableEnumGetter {
+public interface MutableInputTable {
```

Comment:
While you're in this code, can you change the interface from `RowSet` to `TrackingRowSet`?

Also, should we be adding an `add` overload for `allowEdits`? Should we rename that boolean `allowModify`?